docs/D-Modularity/functions-arrays-and-structs.md

```diff
@@ -16,21 +20,21 @@ After reading this section, you will be able to:
 
 Procedural programming scopes information.  Scoping is an essential feature of modular design.  A complete program consists of a variety of scopes.  Each module limits the visibility of the program data and instructions in that module.  Each code block limits the visibility of the data in that block. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/scope.png)
+![Hierarchical view of global, function and block scope](https://ict.senecacollege.ca//~ipc144/pages/images/scope.png)
 
 This chapter describes how to identify a function type, describes the different scopes within a program, describes the passing of arrays and structures to a function, lists style guidelines for function coding and provides a sample walkthrough with functions, pointers and structures. 
 
 ## Prototypes
 
 A function prototype identifies a function type.  It provides the information that the compiler requires to validate a function call.  The prototype is similar to the function header.  A prototype takes the following form
 
-```
+```c
 type identifier(type [parameter], ..., type [parameter]);
 ```
 
 A prototype ends with a semi-colon and may exclude the parameter identifiers.  The identifier, the return type, and the parameter types are sufficient to validate a function call.  The parameter types determine any coercion that may be necessary in passing values to the function. 
```

Comment:
Also here, "The identifier,....", and "The parameter,..."

Reply:
Done